src/VisualStudio/Core/Test.Next/Remote/SerializationValidator.cs

```diff
@@ -28,8 +28,8 @@ private sealed class AssetProvider : AbstractAssetProvider
             public AssetProvider(SerializationValidator validator)
                 => _validator = validator;
 
-            public override Task<T> GetAssetAsync<T>(Checksum checksum, CancellationToken cancellationToken)
-                => _validator.GetValueAsync<T>(checksum);
+            public override async ValueTask<T> GetAssetAsync<T>(Checksum checksum, CancellationToken cancellationToken)
+                => await _validator.GetValueAsync<T>(checksum).ConfigureAwait(false);
```

Comment:
❓why do we await here instead of just returning the value task?

Reply:
because _validator.GetValueAsync returns a Task, nto a ValueTask.  This is a simple way to map between them.